static/css/documentation.css

```diff
@@ -1074,6 +1074,11 @@ a .mega-octicon {
   list-style-type: disc;
 }
 
+.content ul ul{
```

Comment:
add a space between ul and squiggly brace

Reply:
[Space](https://github.com/cerner/fhir.cerner.com/pull/337/commits/51acc559d2f17d86dad80ac82201628a4caaf31b#diff-331344c9aef74ed7c8415f3632a7c619R1077)